### DISCUSSION TITLE
Generador de repos no está funcionando

### DISCUSSION THREAD

**Discussion by JavierMoralesD** [IIC2133-PUC/2024-2#74] [Tarea 1] [answered]
Hola. Al intentar aceptar la tarea en Github Classroom, obtuve un error 500

**Comment by sebacarrascop** [comment, marked as answer]
Hola javier! 
Lamentablemente (y nuevamente) GitHub Classroom está con [problemas](https://github.com/orgs/community/discussions/43783).

~~Se supone que debería arreglarse pronto, por lo que estamos esperando para ver si esto sucede, sino veremos otras opciones.
Deje subido el código base en archivos de Canvas, dentro de la carpeta T1 en caso de que quieras avanzar lo antes posible, (si no se soluciona en un par de horas de todas avisaremos por un anucio de Canvas, no lo hacemos altiro ya que la vez pasada el problema se solucionó a las pocas horas de que ocurrió el error, así que estamos esperando un poco). Saludos!~~

Edit: al minuto que respondí el problema se solucionó, saludos!
